main.cpp

```diff
@@ -12,8 +14,65 @@ Create a branch named Part2
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct Person
+{
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+
+    void run( bool startWithLeftFoot, int howFast );
+};
 
+struct Limb
+{
+    int limbLength;
+    int extremitySize;
 
+    int stepForward();
+    int stepSize();
+};
+
+int Limb::stepForward()
+{
+    int x;
+    bool forward = true;
+    if( forward == true)
+    {
+        x = 1;
+    }
+    else
+    {
+        x = -1;
+    }
+    return x;
+}
+
+int Limb::stepSize()
+{
+    int x = ( limbLength + extremitySize ) * stepForward();
+    return x;
+}
+
+void Person::run( bool startWithLeftFoot, int howFast = 1 )
+{
+    Limb leftFoot;
+    Limb rightFoot;
```

Comment:
These should be member variables in your `Person` class.  They should not be local variables. 

- [ ] move `Limb` instances to inside class definition